src/test/java/org/example/TaskServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package org.example;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TaskServiceTest {
+    TaskService taskService;
+    ArrayList<Task> taskList = new ArrayList<>();
+
+    @Test
+    public void shouldAddTaskToTheList() {
+        Task task1 = new Task("Zadanie 1", false);
```

Comment:
to też jest given

Reply:
jak dałam given przed tym inicjalizacjami, to nie mogłam skompilować, bo czepiało się "is not a statement"